app/database/controller.py

```diff
@@ -1,54 +1,66 @@
 from fastapi import FastAPI,APIRouter
-from app.database.schema import client, Message, db
-from bson.objectid import ObjectId
-some_router = APIRouter() 
+from datetime import datetime
+import dateutil.parser
+from app.database.schema import client, Message, db 
+from app.database.body import Messagebody
+from bson import ObjectId 
+import datetime 
+from tzlocal import get_localzone # $ pip install tzlocal
 
-def get_message():
+
+some_router = APIRouter()
+
+
+def get_message(channel_id,starttime,endtime):
     """
     Args : None 
     
     The Function is used to fetch all the data in the DB 
     
     Return Type : List 
-    
+    {"channel_id":channel_id })
     """
+    starttime = dateutil.parser.parse(starttime)
+    endtime = dateutil.parser.parse(endtime)
+    print(starttime)
+    print(endtime)
     cynergy = []
-    print (db.message.find())
-    for i in db.message.find():
-      print(i)
+    for i in db.message.find({"channel_id":channel_id,'$and':[{'timestamp': { '$gte' : starttime },'timestamp': { '$lt' : endtime }}]}):
       cynergy.append(Message(**i))
-    return {'message is ': cynergy}
+      print(i)
+    return {'The Message Data is ': cynergy}
 
 
-def create_message(name:str,message:str):
-    """
-    Args: 
-    \n Name -> str
-    \n Message -> str
-    
+def create_message(channel_id:str,content:str,sender_id:str):
+    """[summary]
     The Function is used to create only one message and insert the data into DB 
-    
-    Return Type : List 
-    
-    """
-    ret = db.message.insert_one({"Name":name,"Message":message})
-    return {"Name":name,"Message":message}
+    Args:
+        channel_id (str): [description]
+        content (str): [description]
+        sender_id (str): [description]
+        timestamp (str): [description]
 
-def remove_message(_id):
+    Returns:
+        [List]: [description]
     """
-    Args: 
-    \n _id -> String 
+    timestamp = datetime.datetime.now()
+    ret = db.message.insert_one({"channel_id":channel_id,"content":content,"sender_id":sender_id,"timestamp":timestamp})
```

Comment:
always wrap db related update and insertion commands with `try/except` and catch appropriate pymongo error whenever and whereever needed. you should also raise custom errors wherever you seem its necessary to do so and mention the same in docstrings.